src/tree/ScaleRuleGroupTreeItem.ts

```diff
@@ -30,14 +36,39 @@ export class ScaleRuleGroupTreeItem extends AzExtParentTreeItem implements IAzur
         return new ThemeIcon('symbol-constant');
     }
 
+    public async createChildImpl(context: ICreateChildImplContext): Promise<AzExtTreeItem> {
+        const scale: ScaleTreeItem = treeUtils.findNearestParent(this, ScaleTreeItem.prototype) as ScaleTreeItem;
+        const containerApp: ContainerAppTreeItem = treeUtils.findNearestParent(this, ContainerAppTreeItem.prototype) as ContainerAppTreeItem;
+
+        const title: string = localize('addScaleRuleTitle', 'Add Scale Rule');
+        const wizardContext: IAddScaleRuleWizardContext = {
+            ...context, containerApp, scale, scaleRuleGroup: this,
+        };
+        const wizardOptions: IWizardOptions<IAddScaleRuleWizardContext> = {
+            title,
+            promptSteps: [new ScaleRuleNameStep(), new ScaleRuleTypeStep()],
+            executeSteps: [new AddScaleRuleStep()],
+            showLoadingPrompt: true
+        };
+        const wizard: AzureWizard<IAddScaleRuleWizardContext> = new AzureWizard(wizardContext, wizardOptions);
+        await wizard.prompt();
+        context.showCreatingTreeItem(nonNullProp(wizardContext, 'ruleName'));
+        await wizard.execute();
+
+        if (wizardContext.error !== undefined) {
```

Comment:
You should be able to just throw the error from the wizard step instead of tracking it and then throwing it after the execution.